pydm/data_plugins/__init__.py

```diff
@@ -15,6 +16,36 @@
 plugin_modules = {}
 
 
+DEFAULT_PROTOCOL = os.getenv("PYDM_DEFAULT_PROTOCOL")
+if DEFAULT_PROTOCOL is not None:
+    DEFAULT_PROTOCOL = DEFAULT_PROTOCOL.replace('://', '')
```

Comment:
Maybe check `.endswith()` and strip off `[:-3]` instead? I'm imagining someone setting this incorrectly and getting some weird results...